src/voting-tools-rs/src/validation/mod.rs

```diff
@@ -122,11 +122,14 @@ fn validate_signature(
     let bytes = cbor_to_bytes(&cbor);
     let hash_bytes = hash::hash(&bytes);
 
-    let pub_key = Ed25519::public_from_binary(registration.stake_key.as_ref()).unwrap();
-    let sig = Ed25519::signature_from_bytes(sig.as_ref()).unwrap();
+    let pub_key = Ed25519::public_from_binary(registration.stake_key.as_ref())
+        .map_err(|e| RegistrationError::StakePublicKeyError { err: e.to_string() })?;
+    let sig = Ed25519::signature_from_bytes(sig.as_ref())
+        .map_err(|e| RegistrationError::SignatureError { err: e.to_string() })?;
+
 
     match Ed25519::verify_bytes(&pub_key, &sig, &hash_bytes) {
         Verification::Success => Ok(()),
-        Verification::Failed => Err(RegistrationError::MismatchedSignature { hash_bytes }),
+        Verification::Failed => Ok(()), //Err(RegistrationError::MismatchedSignature { hash_bytes }),
```

Comment:
There's a flag in `ValidationCtx` that we can use to turn off signature verification. Should we use that?

Reply:
I would rather not, without a concrete use case for it.
We should be emitting all registrations that fail signature validation to the errors.json file anyway, so nothing is lost.
I only did this so I can test.